src/linpde_gp/randprocs/crosscov/linfunctls/_stacked.py

```diff
@@ -0,0 +1,143 @@
+import numpy as np
+import jax.numpy as jnp
+import probnum as pn
+
+from linpde_gp import linfunctls
+from linpde_gp.linops import BlockMatrix
+
+from .._pv_crosscov import ProcessVectorCrossCovariance
+
+
+@linfunctls.StackedLinearFunctional.__call__.register(  # pylint: disable=no-member
+    ProcessVectorCrossCovariance
+)
+def _(self, pv_crosscov: ProcessVectorCrossCovariance, /) -> np.ndarray:
+    res_L1 = self.L1(pv_crosscov)
+    res_L2 = self.L2(pv_crosscov)
+
+    axis = 0
+    if pv_crosscov.reverse:
+        axis = -1
+    
+    return np.concatenate((res_L1, res_L2), axis=axis)
+
+
+class CovarianceFunction_Identity_Stacked(ProcessVectorCrossCovariance):
+    def __init__(
+        self,
+        covfunc: pn.randprocs.covfuncs.CovarianceFunction,
+        stacked: linfunctls.StackedLinearFunctional,
+    ):
+        self._covfunc = covfunc
+        self._stacked = stacked
+
+        L1 = self._stacked.L1
+        L2 = self._stacked.L2
+
+        self._kL1a = L1(self._covfunc, argnum=1)
+        self._kL2a = L2(self._covfunc, argnum=1)
+        assert isinstance(self._kL1a, ProcessVectorCrossCovariance)
+        assert isinstance(self._kL2a, ProcessVectorCrossCovariance)
+
+        super().__init__(
+            randproc_input_shape=self._covfunc.input_shape,
+            randproc_output_shape=self._covfunc.output_shape_0,
+            randvar_shape=self._stacked.output_shape,
+            reverse=False,
+        )
+
+    @property
+    def kernel(self) -> pn.randprocs.kernels.Kernel:
+        return self._covfunc
+
+    @property
+    def stacked(self) -> linfunctls.StackedLinearFunctional:
+        return self._stacked
+
+    @property
+    def kL1a(self) -> ProcessVectorCrossCovariance:
+        return self._kL1a
+
+    @property
+    def kL2a(self) -> ProcessVectorCrossCovariance:
+        return self._kL2a
+
+    def _evaluate(self, x: np.ndarray) -> np.ndarray:
+        return np.concatenate((self._kL1a(x), self._kL2a(x)), axis=-1)
+
+    def _evaluate_jax(self, x: jnp.ndarray) -> jnp.ndarray:
+        return jnp.concatenate((self._kL1a(x), self._kL2a(x)), axis=-1)
+
+@linfunctls.StackedLinearFunctional.__call__.register(  # pylint: disable=no-member
+    CovarianceFunction_Identity_Stacked
+)
+def _(self, kSa: CovarianceFunction_Identity_Stacked, /) -> pn.linops.LinearOperator:
+    L1kL1a = pn.linops.Matrix(self.L1(kSa.kL1a))
+    L1kL1a.is_symmetric = True
+    L1kL1a.is_positive_definite = True
```

Comment:
I don't think that this is true in general.
Isn't it possible that $k\mathcal{L}^*$ used a different linear functional, i.e. not `L1`?

Reply:
You're right, I didn't consider that.